src/components/product/product.repository.ts

```diff
@@ -105,7 +105,7 @@ export class ProductRepository extends CommonRepository {
             {
               // BC with existing products
               progressStepMeasurement: `"${ProgressMeasurement.Percent}"`,
-              progressTarget: 100,
+              progressTarget: 'props.progressTarget',
```

Comment:
This is for products already existing that don't have `props.progressTarget`. The `merge()` should take the `progressTarget` from `props`, if it exists, instead of this default/fallback.